### ISSUE TITLE
Post 2.0.5 release

### ISSUE THREAD

**Issue by TomAshe** [dcarbone/php-fhir#69]
Hello!

Was wondering if there are any plans for a post 2.0.5 release?

Are there issues with the commits post-2.0.5 preventing a new release?

I was looking to use this (well, actually the php-fhir-generated version), but encountered an issue where Significant Digits are lost for Decimal numbers.

Looks like this was address, though, by this commit last June: https://github.com/dcarbone/php-fhir/commit/2a0461db46e28a5755ff202a5a4481124d96ad37#diff-704bb99ea7436cef0ee4ec7f90a90e15a514727292315c8814d4b2992d2ed5fcR35

Regards!

**Comment by dcarbone** [comment]
@TomAshe: yes, there absolutely is!  Time has gotten the best of me, a bit, and I've been quite busy.

I actually have all next week off work, and I had planned on doing some cleanup items around this repo with the intent of cutting a v3 release with SimpleXML replaced with DOMDocument, as well as a few other improvements.

That specific change, however, is fairly small and I believe it should be easy cut a new release from where v2.0.5 is currently at with that change.

If you feel up to it, and you have time before I can do it, feel free to create a PR and I'll cut a new release!

**Comment by dcarbone** [comment]
@TomAshe: hey Tom, sorry it took so long but I finally got around to backporting the changes necessary to address this problem into the v2.x line.

Please see if release v2.0.8 fixes things for you!
